packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -126,6 +126,51 @@ describe("SharedTree", () => {
 			assert.equal(view.root, 10);
 		});
 
+		it.only("initialize-dispose-view with primitive schema", () => {
+			const tree = treeTestFactory();
+			assert.deepEqual(tree.contentSnapshot().schema.rootFieldSchema, storedEmptyFieldSchema);
+
+			const config = new TreeViewConfiguration({
+				schema: SchemaFactory.number,
+			});
+
+			const view1 = tree.viewWith(config);
+			view1.initialize(10);
+			assert.deepEqual(view1.root, 10);
+
+			view1.dispose();
+
+			const view2 = tree.viewWith(config);
+			assert.deepEqual(view2.root, 10);
+		});
+
+		it.only("initialize-dispose-view with object schema", () => {
```

Comment:
I think we can just merge this change (these are very nice tests for it), but remove the .only on the above one, and .skip on this one with a comment referencing the bug. That will make debugging this a bit easier.